src/spacel/model/app.py

```diff
@@ -24,6 +24,8 @@ def __init__(self, orbit, params=None):
         self.scheme = params.get('scheme', 'internet-facing')
         self.health_check = params.get('health_check', 'TCP:80')
         self.local_health_check = params.get('health_check', 'TCP:80')
+        self.loadbalancer = params.get('loadbalancer', 'true')
+        self.availability = params.get('availability', 'private')
```

Comment:
could rename to:
```
instance_availability=internal||internet-facing
```
as just having `availability` is indeed not very clear, considering there's also a `scheme`